src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -454,10 +454,10 @@
     <value>Toggle command palette</value>
   </data>
   <data name="SuggestionsCommandHistoryCommandKey" xml:space="preserve">
-    <value>Recent commands</value>
+    <value>Recent commands...</value>

```

Comment:
Wait, we have a command just titled "Recent commands"? There's no verb there. Even after this change, it's still verb-less 😅

Reply:
Ah! Looks like this was removed as a part of #15664 (merged 8/15/2023). I'm just gonna remove this key since it's been a while since we've used it.